pkg/oauth/api/types.go

```diff
@@ -167,3 +167,14 @@ type OAuthClientAuthorizationList struct {
 	unversioned.ListMeta
 	Items []OAuthClientAuthorization
 }
+
+type SerializedRedirectReference struct {
```

Comment:
to avoid name clashes, make this `OAuthRedirectReference`, and make the annotation match `oauth-redirectreference`